src/winutil/VariableMenuPopulator.h

```diff
@@ -160,8 +162,17 @@ class WINUTIL_API VariableMenuPopulator
         bool mustRePopMD, bool isSim);
 
 private:
-    typedef std::map<std::string, bool> StringBoolMap;
-    typedef std::map<std::string, std::string> StringStringMap;
+    struct Comparator {
+        bool operator() (const std::string& s1, const std::string& s2) const {
+            std::string str1(s1.length(),' ');
+            std::string str2(s2.length(),' ');
+            std::transform(s1.begin(), s1.end(), str1.begin(), tolower);
+            std::transform(s2.begin(), s2.end(), str2.begin(), tolower);
+            return  str1 < str2;
+        }
+    };
+    typedef std::multimap<std::string, bool, Comparator> StringBoolMap;
+    typedef std::multimap<std::string, std::string, Comparator> StringStringMap;
```

Comment:
There is a StringStringMap defined in common/utility/maptypes.h
Not sure, but your definition may cause problems for a static build?
Are you using your StringStringMap anywhere?

Reply:
I didn't introduce `StringStringMap` typedef here. I just modified its already existing definition from `typedef std::map<std::string, std::string> StringStringMap;`. I am not sure anything defined in `common/utility/maptypes.h` is relevant here as I don't think any of those types are being used here.